<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comComboL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comComboR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushIzberiDat_L">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Izberi datoteko</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zajemiButtonL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zajemi </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Levi kanal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrationButton_L">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kalibracija</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrationButton_R">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kalibracija</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zadnje meritve</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxL">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxR">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Število meritev</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Število meritev</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInstant">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetL">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>min</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>max</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetR">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>min</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>max</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="meritveTable">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKalibracija">
    <property name="title">
     <string>O programu</string>
    </property>
   </widget>
   <addaction name="menuKalibracija"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
